### ISSUE TITLE
ErrImagePull  with zookeeper deployment

### ISSUE THREAD

**Issue by mram0509** [PharosProduction/tutorial-apache-kafka-cluster#1]
Hi,

When I run this command:
kubectl apply -f zookeeper-deployment.yaml && kubectl apply -f zookeeper-ip-service.yaml
I am see the following: 
zookeeper-deployment-84fb6595b8-g6gzn   0/1     ErrImagePull   0          16s

Looks like the image is not available.

**Comment by kenhglee** [comment]
I agree. The error message that I got was "Failed to pull image "pharosproduction/zookeeper_k8s:v4": rpc error: code = Unknown desc = Error response from daemon: manifest for pharosproduction/zookeeper_k8s:v4 not found" upon kubectl describe pod xxx. 

Have replaced the image to "digitalwonderland/zookeeper" from https://dzone.com/articles/ultimate-guide-to-installing-kafka-docker-on-kuber and it worked.
